android/app/src/main/java/com/microsoft/codepush/react/CodePushPackage.java

```diff
@@ -87,7 +104,13 @@ public String getCurrentPackageBundlePath() {
             return null;
         }
 
-        return CodePushUtils.appendPathComponent(packageFolder, UPDATE_BUNDLE_FILE_NAME);
+        WritableMap currentPackage = getCurrentPackage();
```

Comment:
I understand what you're doing here, but I might now in a month :) Could we add a simple comment explaining that the relative path is used for zipped updates vs single file updates?